docs/user/dashboard/tutorial-create-a-dashboard-of-lens-panels.asciidoc

```diff
@@ -66,261 +96,164 @@ image::images/lens_end_to_end_1_3.png[Changed type and dropped clientip field]
 
-. Add the filter for the client error codes. 
+.. In the *Display name* field, enter `Page URL`.
 
-.. Click *Add a filter*.
-
-.. In the *KQL* field, enter `response.keyword>=400 AND response.keyword<500`. 
-
-.. In the *Label* field, enter `4XX`, then press Return.
-
-. Add the filter for the server error codes. 
-
-.. Click *Add a filter*.
-
-.. In the *KQL* field, enter `response.keyword>=500 AND response.keyword<600`. 
-
-.. In the *Label* field, enter `5XX`, then press Return.
-
-. To change the color palette, select *Status* from the *Color palette* dropdown, then click *Close*.
+. Click *Close*.
 
 . Click *Save and return*.
 
-[discrete]
-[[histogram]]
-=== View the traffic for your website by the hour 
-
-To find the best time to shut down your website for maintenance, create a histogram that displays the traffic for your website by the hour.
-
-. Open *Lens*.
+. Arrange the table panel so that it has its own row, but keep the default width of half the screen.
 
-. From the *Available fields* list, drag and drop *bytes* to *Vertical axis* in the editor, then configure the options.
+// The proportion chart types in *Lens* display the percentage of the total,
+// grouped by a function that you choose. Combine these two ideas to see the
+// most frequent operating systems as a percentage:
 
-.. Click *Average of bytes*.
+// *Lens* chooses a default color palette and adds a visual indicator to the dimension that controls
+// the color. To change the color:
 
-.. From *Select a function*, click *Sum*.
+// . In the editor, click on *Top values of machine.os.keyword*, which has a color indicator. 
 
-.. In the *Display name* field, enter `Transferred bytes`.
+// . From the *Color palette* dropdown, select *Compatibility*.
 
-.. From the *Value format* dropdown, select `Bytes (1024)`, then click *Close*.
-
-. From the *Available fields* list, drag and drop *hour_of_day* to *Horizontal axis* in the editor, then configure the options.
-
-.. Click *hour_of_day*.
-
-.. Click and slide the *Intervals granularity* slider until the horizontal axis displays hourly intervals.
-+
-[role="screenshot"]
-image::images/lens_end_to_end_5_2.png[Create custom ranges]
-
-. Click *Save and return*.
+// . Click *Close*.
+// +
+// [role="screenshot"]
+// image::images/lens_end_to_end_2_1.png[Donut chart with open config panel]
 
 [discrete]
 [[custom-ranges]]
-=== View the percent of small versus large transferred files 
- 
-To determine if your users transfer more small files versus large files, create a pie chart that displays the percentage of each size. 
+=== Compare the percentage of a subset to the whole
 
-. Open *Lens*.
+To compare a subset to the whole you need to create non-overlapping sets of documents that add up to 100%.
+The *filters* function is capable of doing this for all data types, but for numeric data the *intervals* function is
+is recommended instead. Use a proportion chart to display the values as a percentage.
```

Comment:
I would expand a bit more on the `proportion` part here, proposing the list of available charts.
```suggestion
is recommended instead. Use a proportion chart to display the values as a percentage: *Lens* offers different types of chart to help with this type of task: pie chart, donut chart, treemap and percentage charts.
```